src/System.Private.ServiceModel/src/System/ServiceModel/Security/WSSecureConversation.cs

```diff
@@ -0,0 +1,599 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+using System;
+using System.Collections.Generic;
+using System.IdentityModel.Selectors;
+using System.IdentityModel.Tokens;
+using System.Runtime;
+using System.ServiceModel;
+using System.ServiceModel.Security.Tokens;
+using System.Xml;
+//using StrEntry = WSSecurityTokenSerializer.StrEntry;
```

Comment:
Issue31 temporary removal or another instance where we're removing for consolidation?